src/api/libcellml/reset.h

```diff
@@ -57,107 +57,112 @@ class LIBCELLML_EXPORT Reset: public OrderedEntity
     VariablePtr variable() const;
 
     /**
-     * @brief Add a child when to this @c Reset.
+     * @brief Set the @c TestVariable for this @c Reset.
      *
-     * Add a copy of the given @p when as a child @c When of this @c Reset.
+     * Sets the @c TestVariable for this @c Reset.
      *
-     * @param when The @c When to add.
+     * @param variable The @c TestVariable to set.
      */
-    void addWhen(const WhenPtr &when);
+    void setTestVariable(const VariablePtr &variable);
 
     /**
-     * @brief Remove the @c When at the given @p index.
+     * @brief Get the @c TestVariable for this @c Reset.
      *
-     * Remove the @c When with the given @p index. @p index must
-     * be in the range [0, \#Whens).
+     * Returns the @c TestVariable of this @c Reset.
      *
-     * @param index The index of the @c When to remove.
-     *
-     * @return True if the @c When was removed, false otherwise.
+     * @return The @c TestVariable.
      */
-    bool removeWhen(size_t index);
+    VariablePtr testVariable() const;
 
     /**
-     * @brief Remove the @c When with the given pointer.
-     *
-     * Remove the @c When with the pointer @p when.
+     * @brief Appends the argument to the testValue string for this reset.
```

Comment:
Replace with something like "@brief Append @p math to the existing testValue string for this @c Reset." (i.e. try to be consistent with what has been done for the `Component` class.) The same for the other API documentation.

Reply:
I think we should do this en masse for all of the documentation everywhere once all the pull requests have been merged.  There are lots of inconsistencies everywhere ... !

Reply:
Agreed, ignore my comment for now.